src/format/opamTypes.mli

```diff
@@ -48,13 +48,33 @@ type std_path =
   | Lib | Bin | Sbin | Share | Doc | Etc | Man
   | Toplevel | Stublibs
 
+(** Download failure explanation *)
+type dl_fail_reason = string option * string
+(** Respectively the short and long version of an error message. The
+    usage is: the first argument is displayed on normal mode (nothing
+    if [None]), and the second one on verbose mode. *)
```

Comment:
Using a record instead here would make it easier to read, use and reason about i think
```suggestion
type dl_fail_reason = {
  short_reason : string option;
  long_reason : string;
}
```